pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -846,6 +846,7 @@ default-package-overrides:
   - ghc-source-gen ==0.3.0.0
   - ghc-syntax-highlighter ==0.0.5.0
   - ghc-tcplugins-extra ==0.4
+  - ghc-tcplugins-extra ==0.3.2
```

Comment:
This should actually go in this list:

https://github.com/NixOS/nixpkgs/blob/89bcf4b7e23f52af03acd81f70cd13deb9b033fd/pkgs/development/haskell-modules/configuration-hackage2nix.yaml#L2356

Also, please rebase this on top of `haskell-updates`.

Reply:
Done. (Was already rebased.)